tdms/src/iterator.cpp

```diff
@@ -6158,72 +5978,6 @@ void mexFunction(int nlhs, mxArray *plhs[], int nrhs, const mxArray *prhs[]) {
   if (exi_present) freeCastMatlab3DArray(exi, *Nt);
   if (eyi_present) freeCastMatlab3DArray(eyi, *Nt);
 
-  //fprintf(stderr,"Pos 18\n");
-  if (dimension == THREE) {

```

Comment:
Is this casting being done elsewhere now, or is it redundant?

Reply:
the `freeX` functions are now being called in the class destructors, so should be okay 👍🏼